source/features/dim-bots.tsx

```diff
@@ -6,9 +6,8 @@ import features from '.';
 
 const commitSelector = [
```

Comment:
The arrays should not be nested, this should be a flat array that is mapped _and then_ joined. (`prSelector` should probably be joined too for consistency.)

Also having a single line for each name would be much more readable.

Reply:
Using `flatMap` on a flat array is pointless :wink: 
What I meant was to remove the outer array.

Reply:
My bad! I'll get the outer array sorted